Trying to build Fuzion on Windows 10 gives an error

I just did git clone and updated Java:
c:\fuzion>javac --version
javac 16.0.2

I installed Make for Windows (make.exe) with: choco install make

GNU Make 4.3
Built for Windows32
Copyright (C) 1988-2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

This is the output I get when running make:

c:\fuzion>make
mkdir -p ./build/classes
javac -source 16 -cp ./build/classes -d ./build/classes ./src/dev/flang/ast/Assign.java ./src/dev/flang/ast/Block.java ./src/dev/flang/ast/BoolConst.java ./src/dev/flang/ast/Box.java ./src/dev/flang/ast/Call.java ./src/dev/flang/ast/Case.java ./src/dev/flang/ast/Check.java ./src/dev/flang/ast/Cond.java ./src/dev/flang/ast/Consts.java ./src/dev/flang/ast/Contract.java ./src/dev/flang/ast/Current.java ./src/dev/flang/ast/Destructure.java ./src/dev/flang/ast/Expr.java ./src/dev/flang/ast/FeErrors.java ./src/dev/flang/ast/Feature.java ./src/dev/flang/ast/FeatureName.java ./src/dev/flang/ast/FeatureVisitor.java ./src/dev/flang/ast/FormalGenerics.java ./src/dev/flang/ast/Function.java ./src/dev/flang/ast/FunctionReturnType.java ./src/dev/flang/ast/Generic.java ./src/dev/flang/ast/If.java ./src/dev/flang/ast/InitArray.java ./src/dev/flang/ast/Impl.java ./src/dev/flang/ast/IncompatibleResultsOnBranches.java ./src/dev/flang/ast/Loop.java ./src/dev/flang/ast/Match.java ./src/dev/flang/ast/Nop.java ./src/dev/flang/ast/NoType.java ./src/dev/flang/ast/NumLiteral.java ./src/dev/flang/ast/Old.java ./src/dev/flang/ast/RefType.java ./src/dev/flang/ast/Resolution.java ./src/dev/flang/ast/ReturnType.java ./src/dev/flang/ast/Stmnt.java ./src/dev/flang/ast/StrConst.java ./src/dev/flang/ast/Tag.java ./src/dev/flang/ast/This.java ./src/dev/flang/ast/Type.java ./src/dev/flang/ast/Types.java ./src/dev/flang/ast/Unbox.java ./src/dev/flang/ast/Universe.java ./src/dev/flang/ast/ValueType.java ./src/dev/flang/ast/Visi.java
.\src\dev\flang\ast\FeatureName.java:184: error: illegal character: '\u2039'
  public int compareTo​(FeatureName o)
                        ^
.\src\dev\flang\ast\FeatureName.java:184: error: ';' expected
  public int compareTo​(FeatureName o)
                                       ^
.\src\dev\flang\ast\FeatureName.java:195: error: illegal character: '\u2039'
  public int compareTo​(Object o)
                        ^
.\src\dev\flang\ast\FeatureName.java:195: error: ';' expected
  public int compareTo​(Object o)
                                  ^
4 errors
make: *** [build/classes/dev/flang/ast/__marker_for_make__] Error 1

_Remark:_
Half a year ago I used the following make.bat to successfully build fuzion (I zipped it for uploading); perhaps this can be useful:
[make.zip](https://github.com/fridis/fuzion/files/7128679/make.zip)

--- a/src/dev/flang/ast/FeatureName.java
+++ b/src/dev/flang/ast/FeatureName.java
@@ -181,7 +181,7 @@ public class FeatureName extends ANY implements Comparable
   /*-----------------------------  methods  -----------------------------*/
 
 
-  public int compareTo​(FeatureName o)
+  public int compareTo(FeatureName o)
   {
     int result = _baseName.compareTo(o._baseName);
     return
@@ -192,7 +192,7 @@ public class FeatureName extends ANY implements Comparable
   }
 
 
-  public int compareTo​(Object o)
+  public int compareTo(Object o)
   {
     return compareTo((FeatureName) o);
   }

